web/packages/teleterm/src/ui/ClusterConnect/ClusterLogin/FormLogin/FormLogin.tsx

```diff
@@ -73,7 +81,7 @@ export default function LoginForm(props: Props) {
 
   if (!localAuthEnabled) {
     return (
-      <FlexBordered p={4}>
+      <FlexBordered px={outermostPadding}>
```

Comment:
I'd add `m={0}` to remove the space at the bottom:
<img width="547" alt="image" src="https://github.com/user-attachments/assets/fbe4fe3c-2dbe-4ff5-b0b7-be8f5b184f72">